src/main/java/uk/gov/companieshouse/officerfiling/api/model/mapper/OfficerFilingMapper.java

```diff
@@ -48,4 +49,12 @@ default String isoDate(Instant instant) {
         }
         return DateTimeFormatter.ISO_LOCAL_DATE.format(instant.atOffset(ZoneOffset.UTC));
     }
+
+    @Mapping(target = "dateOfBirth", source = "dateOfBirth")
+    default String isoDateOfBirth(Date3Tuple tuple) {
+        if (tuple == null) {
+            return null;
+        }
+        return String.format("%d-%d-%d", tuple.getYear(), tuple.getMonth(), tuple.getDay());
```

Comment:
Doesn't the date string need zero padding?
Can do it with the format string `"%04d-%02d-%02d"` or maybe use a standard DateTimeFormatter, e.g.
```return DateTimeFormatter.ISO_DATE.format(LocalDate.of(tuple.getYear(), tuple.getMonth(), tuple.getDay()));```